CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
-## 2.16.0 (Unreleased)
+## 2.16.1 (Unreleased)
```

Comment:
I think we usually mark the Unreleased placeholder as a "minor" rev, i.e., it'd be 2.17.0 in this case.
(that being the most likely next version; it can be changed to 2.16.1 or something else if needed.)

Reply:
Fixed.